.pre-commit-config.yaml

```diff
@@ -22,7 +22,8 @@ repos:
       - id: mypy
         name: mypy
         description: '`mypy` will check Python types for correctness'
-        entry: poetry run mypy
+        entry: poetry run mypy src/puya src/puyapy-stubs test_cases tests examples
```

Comment:
This looks like #106 accidentally made its way here? We definitely don't want to do this because it'll force checking everything on every commit.

Reply:
Without it, `pre-commit run --all` doesn't work on windows, but perhaps we can work around it by adding a poetry script that does the same thing